CONTRIBUTING.md

```diff
@@ -9,7 +7,68 @@ MongoDB welcomes community contributions! If you’re interested in making a con
 
 After the above 2 steps are completed and we've agreed on a path forward:
 1. Fork the repository on GitHub
-1. Create a branch with a name that briefly describes your submission
-1. Add comments around your new code that explain what's happening
-1. Commit and push your changes to your branch then submit a pull request against the current release branch, not master.  The naming scheme of the branch is `release-staging-v#.#.#`. Note: There will only be one release branch at a time.  
-1. A repo maintainer will review the your pull request, and may either request additional changes or merge the pull request.
+2. Create a branch with a name that briefly describes your submission
+3. Add comments around your new code that explain what's happening
+4. Commit and push your changes to your branch then submit a pull request against the current release branch, not master.  The naming scheme of the branch is `release-staging-v#.#.#`. Note: There will only be one release branch at a time.  
+5. A repo maintainer will review the your pull request, and may either request additional changes or merge the pull request.
+
+## Autoclose stale issues and PRs
+- After 30 days of no activity (no comments or commits are on an issue or PR) we automatically tag it as “stale” and add a message: "This issue has gone 30 days without any activity and meets the project’s definition of ‘stale’. This will be auto-closed if there is no new activity over the next 30 days. If the issue is still relevant and active, you can simply comment with a “bump” to keep it open, or add the “[Status] Not Stale” label. Thanks for keeping our repository healthy!"
+- After 30 more days of no activity we automatically close the issue / PR.
+
+## Prerequisites
+- [AWS CloudFormation CLI](https://github.com/aws-cloudformation/cloudformation-cli)
+- (Optional - only need if building from source) [AWS CloudFormation CLI Go Plugin](https://github.com/aws-cloudformation/cloudformation-cli-go-plugin/) > v1.0
+- (Optional - only need if building from source) [Go](https://golang.org/doc/install) v1.19
+
+## Testing the Provider
+Please see README for each resource for details on unit and integrated AWS testing.
+
+## Securing your Deployment
```

Comment:
should this section appear later on the page? We talk about secrets here that users might not have created until the `Securiity - Setup` section

Reply:
Good point, I've moved it later